apps/dapp-console-api/src/utils/getSupportedFaucets.ts

```diff
@@ -0,0 +1,32 @@
+import type { Address } from 'viem'
+
+import { ONCE_UPON_BASE_URL, supportedFaucetConfigs } from '../constants'
+import { envVars } from '../constants/envVars'
+import { Faucet } from '../utils'
+import type { RedisCache } from './redis'
+
+export const getSupportedFaucets = (redisCache: RedisCache) =>
+  supportedFaucetConfigs.map(
+    ({
+      chain,
+      displayName,
+      onChainDripAmount,
+      offChainDripAmount,
+      l1BridgeAddress,
+      isL1Faucet,
+    }) => {
+      return new Faucet({
+        chainId: chain.id,
+        redisCache: redisCache,
+        faucetAddress: envVars.FAUCET_CONTRACT_ADDRESS as Address,
```

Comment:
nit: can remove the `as Address` since the  `envVars.FAUCET_CONTRACT_ADDRESS` type is `Address` already.